bin/copyHelixAssets.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-console */
+/*
+ * @fileOverview Script that copies static assets such as Helix UI CSS and JS from
+ *   node_modules to public/assets folder
+ *
+ *   NOTE: This script require Node.js 8.5 or later
+ *
+ */
+const chalk = require('chalk');
+const fs = require('fs');
```

Comment:
```suggestion
const fs = require('fs-extra');
```

We have [`fs-extra`](https://www.npmjs.com/package/fs-extra) at our disposal that has the same api as `fs` but with some nice convenience functions, too.

Reply:
👍